
geometry_msgs/PoseStamped goal
---
int32 SUCCESS = 0
int32 PREEMPTED = -1
int32 NO_MAP_TO_OBJ_TRANSFORM = -2

int32 error_code
time action_end
---
float32 distance_to_goal
float32 angular_to_goal
time action_start
